src/RGUnityBots/Assets/Scripts/LegacyInputTests/LegacyMouseHandler.cs

```diff
@@ -1,8 +1,34 @@
-﻿using System;
+﻿using UniRx;
+using UniRx.Triggers;
 using UnityEngine;
 
 public class LegacyMouseHandler : MonoBehaviour
 {
+    public void Start()
+    {
+        gameObject.OnMouseDownAsObservable()
+            .SelectMany(_ => this.gameObject.UpdateAsObservable())
+            .TakeUntil(this.gameObject.OnMouseUpAsObservable())
+            .Select(_ => Input.mousePosition)
+            .RepeatUntilDestroy(this) // safety way
+            .Subscribe(
+                x =>
+                {
+                    Debug.Log($"{gameObject.name} OnMouseDownAsObservable()");
```

Comment:
Adds the logging we use in unit tests to confirm that UniRx and mouse inputs work.